packages/webdriverjs/package.json

```diff
@@ -76,10 +76,10 @@
     "typescript": "^4.1.5"
   },
   "dependencies": {
-    "axe-core": "^4.2.0"
+    "axe-core": "^4.2.1"
   },
   "peerDependencies": {
-    "selenium-webdriver": ">= 2.53.1"
+    "selenium-webdriver": "<= 4.0.0-beta.3 || >= 2.53.1"
```

Comment:
Was this change intentional? Seems unrelated.

Reply:
This was intentional to remove the warning message since we use a `prerelease` version of selenium-webdriver.

Reply:
This might be a little wider than you want; I think the `<= 4.0.0-beta.3` part would accept v1.0.0.

Reply:
(you can test this locally using the same library npm uses for semver matching with `npx semver -r "<= 4.0.0-beta.3 || >= 2.53.1" "1.0.0"` - it outputs `1.0.0`, which indicates that it believes `1.0.0` matches the given range)